docs/src/playwright-test-versus-library-js.md

```diff
@@ -0,0 +1,127 @@
+---
+id: playwright-test-vs-library
+title: "Playwright Test vs. Library"
+---
+
+
+Playwright Library provides unified APIs for launching and interacting with browsers, while Playwright Test provides all this plus a fully managed end-to-end Test Runner and experience.
+
+Under most circumstances, for end-to-end testing, you'll want to use `@playwright/test` (Playwright Test), and not `playwright` (Playwright Library) directly. If you are integrating Playwright with a third-party test runner, or are working on scraping/automation, you will use Playwright Library directly.
```

Comment:
I'm not sure yet how a user arrives at this doc; however, if you get here, I want to quickly get the user out and back the testing docs unless they *really* want to use the lib directly and therefore need this info.